### PR TITLE
data: chaitali-khangar

### PR DESCRIPTION
**Pull request by chaitali-khangar**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7468"></a>

### CODE REVIEW

File: data/chaitali-khangar.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Chaitali Khangar",
+  "bio": "I'm a Ruby on Rails and Go lang developer with a passion for mentoring.",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "avatar": "https://github.com/chaitali-khangar.png",
```
Comment (amandamartin-dev) [inline data/chaitali-khangar.json:6]: avatar is no longer needed
```suggestion
```

File: data/chaitali-khangar.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Chaitali Khangar",
+  "bio": "I'm a Ruby on Rails and Go lang developer with a passion for mentoring.",
+  "displayStatsPublic": true,
```
Comment (amandamartin-dev) [inline data/chaitali-khangar.json:4]: stats is not needed
```suggestion
```

**Review by amandamartin-dev — CHANGES_REQUESTED** [review]

**Comment by chaitali-khangar** [comment]
Hi @amandamartin-dev Thank you for pointing it out.

File: data/chaitali-khangar.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Chaitali Khangar",
+  "bio": "I'm a Ruby on Rails and Go lang developer with a passion for mentoring.",
+  "type": "personal",
```
Comment (kumarsonsoff3) [inline data/chaitali-khangar.json:4]: We no longer need the `type` key as we only accept personal profiles by default now. Please remove this from your `json` .

```suggestion
```

**Review by kumarsonsoff3 — DISMISSED** [review]

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**